src/navigation/drive.py

```diff
@@ -152,6 +186,14 @@ def get_drive_command(
         rover_pos = rover_pose.position
         rover_pos[2] = 0
         target_pos[2] = 0
+
+        if np.linalg.norm(target_pos - rover_pos) < completion_thresh:
+            self.reset()
+            return (Twist(), True)
```

Comment:
I'm assuming this sends a zero command to the drive train, why don't we just keep driving towards the target but also return true? It seems to me like this may be causing more choppiness in the search pattern

Reply:
this is the drive function thats used for everything so we can't just change it here, I could pass an additional variable with like minimum speed or something and maybe set it to the minimum speed which is defaulted to zero and for the search we can send some positive value.

Reply:
Gotcha. I don't think it's worth making that change right now, the new spiral seems to be behaving fine.